frontend/src/models/game.js

```diff
@@ -72,4 +74,10 @@ export class Game {
   addScore(score) {
     this.scorecard.addScore(score.winner(), score.points());
   }
+
+  forceComputer() {
+    if (options.autoNextTrick) {
+      this.currentRound.nextMove();
+    }
+  }
```

Comment:
Wondering if we can auto-detect this behavior. If all players in the game are computer players, this is the desired behavior for the game. Instead of checking for `autoNextTrick` we could iterate all players and if none of them is marked as human, trigger `nextMove()`.